.circleci/config.yml

```diff
@@ -112,6 +112,9 @@ jobs:
       gke_project_id:
         type: string
         default: "ci-management"
+      namespace:
+        type: string
+        default: ""
```

Comment:
I prefer if the default value is more explicit. I guess `default: "default"` is better here.